cypress/integration/view_warning_note_details_child.ts

```diff
@@ -0,0 +1,36 @@
+describe('Children Group', () => {
+  beforeEach(() => {
+    cy.getCookies().should('be.empty');
+    cy.setCookie('hackneyToken', Cypress.env('TEST_KEY_CHILDREN_GROUP'));
+    cy.getCookie('hackneyToken').should(
+      'have.property',
+      'value',
+      Cypress.env('TEST_KEY_CHILDREN_GROUP')
+    );
+    cy.visit(
```

Comment:
Same here you can now use `cy.visitAs` now it's merged 😄

Reply:
Done in b98d427 🎉